endpoints.json

```diff
@@ -25,5 +25,6 @@
         }
       ]
     }
-  }
+  },
+  "Things to include": "description, acceptable queries, format and example as above"
```

Comment:
Nice, but changes like this shouldnt be pushed up to github. If you would like to make notes. I would recommend creating a notes file and adding it to your .gitignore. That way you can keep your notes locally without worring about them accidentally going up to github.